src/go/k8s/pkg/resources/configmap.go

```diff
@@ -286,8 +286,8 @@ func (r *ConfigMapResource) createConfiguration(
 		cr.Other = make(map[string]interface{})
 	}
 	cr.Other["auto_create_topics_enabled"] = r.pandaCluster.Spec.Configuration.AutoCreateTopics
-	cr.Other["enable_idempotence"] = true
-	cr.Other["enable_transactions"] = true
+	cr.Other["enable_idempotence"] = false
```

Comment:
I thought idempotence was GA. Where can we check that?

Reply:
@vsaraswat confirmed that idempotence will be released in `v22.1.1` when the some core feature lands.